From 96a9ae961337c6f20611ea8cf92091b3adf9fb8d Mon Sep 17 00:00:00 2001
From: Elon Zhang <zhangzj@rock-chips.com>
Date: Sat, 28 Mar 2020 11:21:55 +0800
Subject: [PATCH] ARM: dts: rockchip: rv11xx-evb-v10: add optee node

Optee node is default disabled. If you need to run
trusted application, this node should be set as okay.

Signed-off-by: Elon Zhang <zhangzj@rock-chips.com>
Change-Id: I0ec40d3f6bdcdad42565c72a17d7cbb9c30ec555
---
 arch/arm/boot/dts/rv11xx-evb-v10.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
index 82a0914840ab..1dca0ccf668c 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
+++ b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
@@ -791,6 +791,10 @@
 	status = "okay";
 };
 
+&optee {
+	status = "disabled";
+};
+
 &pdm {
 	status = "okay";
 	#sound-dai-cells = <0>;
-- 
2.35.3

